Add documentation for passing params to action show page (#210)
avo-hq/docs.avohq.io: docs/3.0/actions.md

end
```

## Passing Params to the Action Show Page

> When working with resource index or show views, it's often necessary to pass parameters to an action.

Reviewer: ```suggestion
When navigation to an action from a resource <Index /> or <Show /> views, it's sometimes useful to pass parameters to an action.
```

> Example

Reviewer: I don't think we need the "Example" copy if we have "consider the following example" (which is great copy! GJ!)

Author: Ok, I will delete it. Thanks for the review.

Consider the following scenario:

> 1. Navigate to http://localhost:3030/admin/resources/users?hey=ya.

Reviewer: Let's use example.com instead of localhost please

2. Attempt to run the dummy action.
You should be able to access the `hey` parameter and retrieve the value `ya`.

#### Implementation

> To achieve this, we'll utilize the `request.referer` object and extract parameters from the URL. Here is how to do it:

Reviewer: ```suggestion
To achieve this, we'll reference the `request.referer` object and extract parameters from the URL. Here is how to do it:
```

end
```

## Passing Params to the Action Show Page
When working with resource index or show views, it's often necessary to pass parameters to an action.

One particular example is when you'd like to populate a field in that action with some particular value based on that param.

```ruby
class Action
  def fields
    field :some_field, as: :hidden, default: -> { if previous_param == yes ? :yes : :no}
  end
end
```
Example

Consider the following scenario:

1. Navigate to http://localhost:3030/admin/resources/users?hey=ya.
2. Attempt to run the dummy action.
You should be able to access the `hey` parameter and retrieve the value `ya`.

> 

Reviewer: Please, let's split this line into multiple lines and statements to make it easier to understand.

Author: ok.

#### Implementation

To achieve this, we'll utilize the `request.referer` object and extract parameters from the URL. Here is how to do it:

```ruby
class Action
  def fields
    field :some_field, as: :hidden, default: -> {
      parent_params = URI.parse(request.referer).query.split("&").map { |param| param.split("=")}.to_h.with_indifferent_access

>       if parent_params[:hey] == 'ya'

Reviewer: Let's add these as comments above each code line that we'll have in the code example.

Author: Ok
